docs/src/components/code-block/examples.tsx

```diff
@@ -30,25 +30,23 @@ export interface ExampleData {
 
 interface ExamplesSwitcherProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
similar comment about props and not need to extend the element attributes if we aren't going to pass them to the underlying dom node